modules/adding-node-iso-configs.adoc

```diff
@@ -68,6 +68,14 @@ For more information, see "Root device hints" in the "Setting up the environment
 The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
 |A dictionary of host network configuration objects.
 
+|cpuArchitecture
+|Specifies the architecture of the nodes you are adding.
+|String
+
+|sshKey
+|The file containing the SSH key to authenticate access to your cluster machines.
```

Comment:
Should this mention a filepath to the file, or anything about where the file must be located if you do specify a key?

Reply:
It sounds ok as it is. As per the other field, also this one must be marked as optional